src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static Random random = new Random();
+    private static Color color;
+
+    public static Ball getRandomBall() {
+        int number = random.nextInt(100);

```

Comment:
Please introduce constant with proper naming
https://github.com/mate-academy/jv-lottery/blob/master/checklist.md#all-magic-numbers-in-your-code-should-be-constants